pkg/controller/installation/installation_controller.go

```diff
@@ -121,6 +140,7 @@ func createInstallationCR(ctx context.Context, serverClient k8sclient.Client) er
 				Type:            string(integreatlyv1alpha1.InstallationTypeManaged),
 				NamespacePrefix: DefaultInstallationPrefix,
 				SelfSignedCerts: false,
+				SMTPSecret:      "rhmi-smtp",
```

Comment:
To confirm, if the RHMI operator is left to create it's own Installation CR. The secret will be expected to be called `rhmi-smtp` and will be expected to be in the same namespace as the RHMI operator?

Just confirming for SOP work

Reply:
Yes :+1: 

Also, currently the secret will always be expected to be in the same namespace as the operator/CR, no matter what it's named.